Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -90,6 +90,20 @@ def run
           end
         end
 
+        # Skip packages that are autobumped by BrewTestBot, if there are any.
+        if autobump_core_path.exist? && autobump_cask_path.exist?
+          autobump_core = File.read(autobump_core_path).lines.map(&:strip)
+          autobump_cask = File.read(autobump_cask_path).lines.map(&:strip)
+
+          formulae_and_casks_to_check = formulae_and_casks_to_check.reject do |formula_or_cask|
+            name = formula_or_cask.respond_to?(:token) ? formula_or_cask.token : formula_or_cask.name
+            if (autobump_core + autobump_cask).include?(name)
```

Comment:
I don't think this logic will work as intended if a formula and cask both share the same name and the formula/cask is only in one of the autobump lists. For example, if the cask is in its autobump list but the formula isn't, the formula would still be skipped like the cask. Instead, we should only be checking the autobump file from the `formula_or_cask` tap.

I could be wrong but it may be technically possible for a tap other than core/cask to have an `autobump.txt` file and it seems like we would want to respect that in the same way. Either way, we could take a more generic approach instead of special-casing first-party taps. You may be able to borrow some ideas from [`Livecheck::load_other_tap_strategies`](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/livecheck/livecheck.rb#L46-L62) (omitting the logic to skip first-party taps, of course).

Reply:
Ah, thanks for reminding me of packages with the same name in both core and cask!